docs/application/native/guides/text-input/tts.md

```diff
@@ -686,6 +719,55 @@ To start, pause, and stop the playback, follow the steps below:
   }
   ```
 
+<a name="control"></a>
+## Playing Mode
+
+It is possible to determine whether to request playback of PCM data synthesized by the TTS service or to receive and play it directly by the TTS client.
+
+To decide between the client-side playback mode and the service-side playback mode, follow the steps below:
+
+- Set the playing mode
+
+  If the application wants to play the PCM data directly instead of playing it that the application requested for synthesis in the TTS service, you can set the playback mode to 'TTS_PLAYING_MODE_BY_CLIENT'. If not set, because the default value is 'TTS_PLAYING_MODE_BY_SERVICE', the TTS service will play it when the TTS client requests to synthesize the text.
+
+  > [!NOTE]
+  > If the playing mode is 'TTS_PLAYING_MODE_BY_SERVICE', you don't need to set callback. It will be played automatically by the TTS service.
+
+  ```cpp
+  void
+  set_playing_mode(tts_h tts)
+  {
+      int ret;
+      int playing_mode = TTS_PLAYING_MODE_BY_CLIENT;
+      ret = tts_set_playing_mode(tts, playing_mode);
+      if (TTS_ERROR_NONE != ret)
+          /* Error handling */
+  }
+  ```
+
+- Receive the synthesized pcm data
```

Comment:
This does not seem to be neccessary, because this description and sample code is already in "Set and unset callbacks" section.

Reply:
Hmm... Yes, you're right. Actually, this is duplicated with "Set and unset callbacks". But I want to explain how connects the relation between the playing mode function and the callback function. and I also intend to explain how the synthesized PCM callback can receive the PCM data after setting playback mode. 
Anyway, if you think that this is necessary, I think that it is O.K. to remove them. :)

Reply:
I agree that guide should explain the relation. However, I don't think it shows the relation to explain again the way to set the callback. And I don't think that playing mode is directly related with the callback. but actually I think that the specific mode `TTS_PLAYING_MODE_BY_CLIENT` is related with the callback.
So, I think that it is possible to describe this situation using simple note like this.
```suggestion
> [!NOTE]
> If you set the mode as TTS_PLAYING_MODE_BY_CLIENT, you should set the synthesized PCM callback because the synthesized PCM data is transferred by the callback.
```

I belive there should be better way to describe it.

Reply:
Thank you for sharing your opinion. I agree with yours. 
I've just removed the duplicated explanation and add it in to the note.